lib/livebook/secrets/secret.ex

```diff
@@ -5,18 +5,20 @@ defmodule Livebook.Secrets.Secret do
   @type t :: %__MODULE__{
           name: String.t(),
           value: String.t(),
-          hub_id: String.t() | nil
+          hub_id: String.t() | nil,
+          deployment_group_id: String.t() | nil
```

Comment:
I think we can remove this now, right? They will always be nested inside a deployment_group_id and, when creating them in the form, we will know the deployment group id because of the page we are currently on?